src/prometheus_alert_rules/reboot.rules

```diff
@@ -0,0 +1,14 @@
+groups:
+- name: HostReboot
+  rules:
+  - alert: HostReboot
+    expr: changes(node_boot_time_seconds[1m]) > 0
+    for: 1m
```

Comment:
I played a bit with `changes(node_boot_time_seconds[...])` on https://demo.promlens.com/ and I didn't get the results I expected. Just making sure that you're getting what you're expecting.

Also:
- `node_boot_time_seconds[1m]` may match the scrape interval, so `changes` may always be zero. Would `5m` make more sense?
- Should the duration be `for: 0m` instead? Combined with 1m above, after 1m it's going to be zero again.

Reply:
Yes you are right, it was not working properly. My goal here was to fire alert rule only once on reboot and not multiple times.

Reply:
I updated alert rule and tests, see PR description.